eng/versioning/version_client.txt

```diff
@@ -118,7 +118,7 @@ com.azure:azure-data-appconfiguration-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-data-schemaregistry;1.4.3;1.5.0-beta.1
 com.azure:azure-data-schemaregistry-apacheavro;1.1.14;1.2.0-beta.2
 com.azure:azure-data-schemaregistry-jsonschema;1.0.0-beta.1;1.0.0-beta.2
-com.azure:azure-data-tables;12.3.19;12.4.0-beta.1
+com.azure:azure-data-tables;12.3.20;12.4.0-beta.1
```

Comment:
NIT: Do you want to update version in the README as well?

Reply:
Oh yes, thank you for reminding me.